dashboard/pubspec.yaml

```diff
@@ -30,7 +30,7 @@ dependencies:
   protobuf: ^2.0.1
   provider: ^6.0.2
   url_launcher: ^6.0.6
-  flutter_app_icons: ^0.0.2
+  flutter_app_icons: ^0.0.3
```

Comment:
When I go to pub.dev's site for this, it redirects to https://github.com/codebird/flutter_app_icons for the source code which is an empty repo. Can you upload the source code?

Reply:
Change to master branch

Reply:
Updated default branch